traitcuration/traits/datasources/ols.py

```diff
@@ -0,0 +1,36 @@
+import requests
+
+
+BASE_URL = "https://www.ebi.ac.uk/ols/api/"
+
+
+def make_ols_query(term_uri, ontology):
+    """
+    Takes in a term URI (or IRI as referenced in OLS documentation) and the ontology id to search against. Returns a
+    dictionary for that term with the fields 'label', 'obo_id' which is the CURIE and 'status'.
+    """
+    results = requests.get(
+        f"{BASE_URL}ontologies/{ontology}/terms?iri={term_uri}").json()
+    print(f"{BASE_URL}ontologies/{ontology}/terms?iri={term_uri}")
+    if 'error' in results:
+        return None
+    term_info = results["_embedded"]["terms"][0]
+    term_curie = term_info["obo_id"]
+    term_label = term_info["label"]
+    term_status = 'current' if ontology == 'efo' else 'needs_import'
```

Comment:
It looks to me that this is not the best place to decide the final term status. I get the logic (you first call EFO, and other ontologies are queried _only if_ nothing was found in EFO), but it's better to make it separate and explicit. That is, `make_ols_query` should only return whether the requested term is in the requested ontology; and the final trait status calculations should be done outside of it in `zooma.py`